CONTRIBUTING.md

```diff
@@ -88,6 +88,8 @@ For more information on github 'force push' workflows see [here] (http://blog.ad
 
 It is perfectly fine for your Pull Request to contain more than one patch - use as many patches as you need to implement the Request (see the previously mentioned 'small patch' thoughts). Each Pull Request should only cover one topic - if you mix up different items in your patches or pull requests then you will most likely be asked to rework them.
 
+All PRs must contain a reference to the issue(s) they resolve (see #Closing-issues).
```

Comment:
I think you can use [Closing issues](CONTRIBUTING.md#closing-issues) here for a hotlink - but, I'm not going to let that stop a merge - your call.